src/neo4j_graphrag/llm/ollama_llm.py

```diff
@@ -0,0 +1,76 @@
+#  Copyright (c) "Neo4j"
+#  Neo4j Sweden AB [https://neo4j.com]
+#  #
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  #
+#      https://www.apache.org/licenses/LICENSE-2.0
+#  #
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+from typing import Any, Optional
+
+from neo4j_graphrag.exceptions import LLMGenerationError
+
+from .base import LLMInterface
+from .types import LLMResponse
+
+
+class OllamaLLM(LLMInterface):
```

Comment:
LGTM but do we not want some tests for this class like test_openai_llm.py?

Reply:
Right, will add some tests.

Reply:
Done in https://github.com/neo4j/neo4j-graphrag-python/pull/231/commits/63d0ad67543df056b2e8abd9cecdcbe95aaf6575